### DISCUSSION TITLE
how to  compute metric volume_size from GB to TB

### DISCUSSION THREAD

**Discussion by leejshades** [NetApp/harvest#3014] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
compute metric volume_size from bytes to TB
![image](https://github.com/NetApp/harvest/assets/7189444/be7deedf-4d97-4646-8835-4ddbe9e9acf9)

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by Hardikl** [comment, marked as answer]
@leejshades 

There are multiple ways to do this conversion, conversion in Prometheus and Grafana would be quite easy. 
As you are interested in Harvest side, This would be the way we can achieve the same in Harvest side.

Current `volume_size` value of one of the volume:
<img width="1780" alt="image" src="https://github.com/NetApp/harvest/assets/83282894/297f35b8-0985-4c1e-b167-633fec120576">

<img width="1777" alt="image" src="https://github.com/NetApp/harvest/assets/83282894/a4d66fb9-ec67-4a1c-ab2f-096cb0a9f887">


<img width="1736" alt="image" src="https://github.com/NetApp/harvest/assets/83282894/e59a519a-1fdb-4b60-bbce-4b5c592b148f">

Created new metric in template as `size_TB` in compute_metric as mentioned in [Doc Detail](https://netapp.github.io/harvest/24.05/plugins/#compute_metric)
<img width="999" alt="image" src="https://github.com/NetApp/harvest/assets/83282894/37128c1d-a520-4e32-832d-922ae2e20cdb">

`size_TB` Result:
<img width="1767" alt="image" src="https://github.com/NetApp/harvest/assets/83282894/ba8be656-7491-4770-ae0b-ed42595dec1e">
